scipy/signal/tests/test_cont2discrete.py

```diff
@@ -294,9 +294,8 @@ def u(t):
         dt = t[1] - t[0]
         u1 = u(t)
 
-        # Use lsim2 to compute the solution to the continuous system.
-        t, yout, xout = lsim2((a, b, c, d), T=t, U=u1, X0=x0,
-                              rtol=1e-9, atol=1e-11)
+        # Use lsim to compute the solution to the continuous system.
+        t, yout, xout = lsim((a, b, c, d), T=t, U=u1, X0=x0)
```

Comment:
So IIUC, we we're passing these kwargs to `odeint` before. The new method should be better(?), but by removing the possibility to pass in keywords, we'll hard-break anyone who currently uses `kwargs`. I think we'll have to add something to `lsim` to warn on unknown keywords (and deprecate them).

Reply:
Is this necessary? The user has to adopt the code anyway and switch to the new function, so removing the `**kwargs` would be possible at the same step.

Reply:
Yes. I think it would be good to add something to the `lsim2` docstring about how to move such cases (that are using kwargs) to something equivalent with `lsim`